code/modules/mob/living/carbon/xenomorph/abilities/queen/queen_powers.dm

```diff
@@ -353,7 +353,18 @@
 	user_xeno.hive.stored_larva--
 	return ..()
 
-/datum/action/xeno_action/onclick/banish/use_ability(atom/Atom)
+/datum/action/xeno_action/onclick/ManageHive/use_ability(atom/Atom)
+	var/mob/living/carbon/xenomorph/queen/queenbanish = owner
+
+
+
+	var/choice = tgui_input_list(queenbanish, "Manage The Hive", "Hive Management",  list("Banish", "Re-Admit", "De-evolve"), theme="hive_status")
+	switch(choice)
+		if("Banish") banish(target)
+		if("Re-Admit") Readmit(target)
+		if("De-evolve") Devolve(target)
```

Comment:
Don't execute code on the same line as the condition:
https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#control-statements